host/sgx/create.c

```diff
@@ -822,7 +848,13 @@ oe_result_t oe_sgx_build_enclave(
 
 #ifdef OE_WITH_EXPERIMENTAL_EEID
     OE_CHECK(_add_eeid_marker_page(
-        context, enclave, image_size, oeimage.elf.entry_rva, &props, &vaddr));
+        &oeimage,
+        context,
+        enclave,
+        image_size,
+        oeimage.elf.entry_rva,
```

Comment:
@volcano0dr looks like a bit of leftover code from a previous iteration. Once you fix this and Dave's comments, I can kick off a CI run, thanks!

```suggestion
        context,
        enclave,
        image_size,
        oeimage.elf.entry_rva,
        num_tls_pages,
```

Reply:
This error has been corrected and the code has been modified according to Dave's comment.
53dd6448d1555101d2fd6a3ec0dc17e4c1fa78ef